stuff/run-php-tests.sh

```diff
@@ -0,0 +1,29 @@
+set -e
+
+# This script is the one which actually runs the PHPUnit test suite.
+# In its default usage:
+#   stuff/run-php-tests.sh
+#
+# it runs all tests in the 'frontend/tests' directory.
+#
+# However, this script will also pass along any arguments to the phpunit
+# command, so one can run specific tests and use phpunit's filtering
+# capabilities.
+#
+# For example:
+#   stuff/run-php-tests.sh frontend/tests/controllers/UserRankTest.php \
+#       --filter testUserRankingClassName
+
+OMEGAUP_ROOT=$(/usr/bin/git rev-parse --show-toplevel)
+
+if [[ -z "$@" ]]; then
+        ARGS="${OMEGAUP_ROOT}/frontend/tests/";
+else
+        ARGS="$@";
+fi
+
+"${OMEGAUP_ROOT}/vendor/bin/phpunit" \
```

Comment:
```suggestion
exec "${OMEGAUP_ROOT}/vendor/bin/phpunit" \
```

así tenemos un proceso menos en el árbol